### DISCUSSION TITLE
Custom written rules support

### DISCUSSION THREAD

**Discussion by vnecas** [super-linter/super-linter#2511] [Ideas]
Hello,
I would like to ask, if here is support for a custom rule, which is written manually. Eg. I would like to add a rule, which disables to use one line comment (starting `//`).

Is there some way how to do that?

Thank you!
Best regards,
Vojta

**Comment by ferrarimarco** [comment]
There's not enough context to answer your question :)

Closing this one because it's quite old, but feel free to open a new issue or discussion.
